pkg/testdata/models/apis/ec2/0000-00-00/generator.yaml

```diff
@@ -0,0 +1,60 @@
+ignore:
+  resource_names:
+    - AccountAttribute
+    - CapacityReservation
+    - CarrierGateway
+    - ClientVpnEndpoint
+    - ClientVpnRoute
+    - CustomerGateway
+    - DefaultSubnet
+    - DefaultVpc
+    - EgressOnlyInternetGateway
+    - Fleet
+    - FpgaImage
+    - Image
+    - Instance
+    - InstanceExportTask
+    - InternetGateway
+    - KeyPair
+    - LaunchTemplateVersion
+    # - LaunchTemplate
```

Comment:
did you mean to not ignore launch template?

Reply:
Yes, there was an existing unit test for EC2 launch templates. Figured I'd pop this generator in there, and ignore all other resources.